packages/jest-environment-jsdom/package.json

```diff
@@ -8,13 +8,15 @@
   },
   "license": "MIT",
   "main": "build/index.js",
+  "types": "build/index.d.ts",
   "dependencies": {
+    "@jest/environment": "^24.1.0",
     "@jest/fake-timers": "^24.1.0",
+    "@jest/types": "^24.1.0",
     "jest-mock": "^24.0.0",
-    "jest-util": "^24.0.0",
-    "jsdom": "^11.5.1"
```

Comment:
keep this

Reply:
Oh, sure! That's a typo. Thanks!

Reply:
Haven't reviewed my own code yet. Just made a PR to ask questions :)